packages/@lightningjs/ui-components/src/components/Card/CardRadio.d.ts

```diff
@@ -17,21 +17,46 @@
  */
 
 import lng from '@lightningjs/core';
-import type { StylePartial } from '../../types/lui';
+import { StylePartial } from '../../types/lui';
 import { TextBoxStyle } from '../TextBox';
 import CardTitle, { CardTitleStyle } from './CardTitle';
-import type { TextContent } from '../InlineContent/InlineContent';
+import { TextContent } from '../InlineContent/InlineContent';
 
-export type CardRadioStyle = CardTitleStyle & {
+type CardRadioStyle = CardTitleStyle & {
   descriptionTextStyle: TextBoxStyle;
   subtitleTextStyle: TextBoxStyle;
 };
-export default class CardRadio extends CardTitle {
+
+declare namespace CardRadio {
+  export interface TemplateSpec extends CardTitle {
+    /**
+     * Object containing all properties supported in the Radio Component
+     */
+    radio?: Record<string, unknown>;
```

Comment:
we should flag this to revisit once we've updated `Radio`, we may be able to provide a better type here

Reply:
Added a TODO and added a note to the Radio Jira ticket to check this